jerry-core/include/jerryscript-compiler.h

```diff
@@ -45,7 +45,17 @@ extern "C"
 #define JERRY_LIKELY(x) __builtin_expect(!!(x), 1)
 #define JERRY_UNLIKELY(x) __builtin_expect(!!(x), 0)
 
-#endif /* __GNUC__ */
+#endif /* !__GNUC__ */
+
+#ifdef _MSC_VER
+void * __cdecl _alloca (size_t _Size);
+
+#define JERRY_ATTR_DEPRECATED __declspec(deprecated)
+#define JERRY_ATTR_NOINLINE __declspec(noinline)
+#define JERRY_ATTR_NORETURN __declspec(noreturn)
+
+#define JERRY_VLA(type, name, size) type *name = (type *) (_alloca (sizeof (type) * size))
+#endif /* !_MSC_VER */
```

Comment:
This update has brought in unintended changes. The comments after `#endif`s "repeat" the condition. Here, and above at GCC's conditional block, the condition is whether `__GNUC__` or `_MSC_VER` is defined, so that's what should be repeated at the corresponding endifs. Below, however, the condition is whether JERRY_VLA is _not_ defined, so the endif's comment is `!JERRY_VLA`.